specification/migrate/resource-manager/Microsoft.Migrate/stable/2019-05-01/migrate.json

```diff
@@ -0,0 +1,3778 @@
+      "description": "An error response from the Azure Migrate service."
+    },
+    "CollectorBodyAgentSpnProperties": {
+      "type": "object",
+      "properties": {
+        "authority": {
+          "description": "AAD Authority URL which was used to request the token for the service principal.",
+          "type": "string"
+        },
+        "applicationId": {
+          "description": "Application/client Id for the service principal with which the on-premise management/data plane components would communicate with our Azure services.",
+          "type": "string"
+        },
+        "audience": {
+          "description": "Intended audience for the service principal.",
+          "type": "string"
+        },
+        "objectId": {
+          "description": "Object Id of the service principal with which the on-premise management/data plane components would communicate with our Azure services.",
+          "type": "string"
+        },
+        "tenantId": {
+          "description": "Tenant Id for the service principal with which the on-premise management/data plane components would communicate with our Azure services.",
+          "type": "string"
+        }
+      }
+    },
+    "HyperVCollector": {
+      "type": "object",
+      "properties": {
+        "eTag": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/CollectorProperties"
+        }
+      }
+    },
+    "CollectorProperties": {
+      "type": "object",
+      "properties": {
+        "discoverySiteId": {
+          "type": "string"
+        },
+        "createdTimestamp": {
+          "type": "string",
+          "readOnly": true
+        },
+        "updatedTimestamp": {
+          "type": "string",
+          "readOnly": true
+        },
+        "agentProperties": {
+          "$ref": "#/definitions/CollectorAgentProperties"
+        }
+      }
+    },
+    "CollectorAgentProperties": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true
+        },
+        "version": {
+          "type": "string",
+          "readOnly": true
+        },
+        "lastHeartbeat": {
+          "type": "string",
+          "format": "date-time",
+          "x-nullable": true,
+          "readOnly": true
+        },
+        "spnDetails": {
+          "$ref": "#/definitions/CollectorBodyAgentSpnProperties"
+        }
+      }
+    },
+    "VMwareCollector": {
+      "type": "object",
+      "properties": {
+        "eTag": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "read
```

Comment:
consider making this "listDownloadUrl".  I know it sounds odd but it lets people consume it from a template deployment.  Existing example:  https://docs.microsoft.com/en-us/rest/api/containerregistry/runs/getlogsasurl

Reply:
Actually, we will always be having a single URL so we are not thinking of making it as a list. Moreover changing this now will lead to a lot of breaking changes for us. So it would be fine for now to keep it as is.